hazelcast/src/main/java/com/hazelcast/map/impl/client/MapGetRequest.java

```diff
@@ -126,9 +126,6 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
```

Comment:
I think the only issue is if you are doing security filtering by the async method, but not the sync one, which I can't see why someone would want to do. The security filtering should be done based on the actual operation being done, not what method name is being called in the client. This is even more true when we have multiple clients and not just Java.